scripts/generate_openapi_document.py

```diff
@@ -22,7 +22,8 @@ def main():
                                atlas='hca',
                                internal=False,
                                plugins=dict(metadata=config.Catalog.Plugin(name='hca'),
-                                            repository=config.Catalog.Plugin(name='tdr')))
+                                            repository=config.Catalog.Plugin(name='tdr')),
+                               sources=set(''))
```

Comment:
This constructs an empty set. Be sure to understand why. If you really want an empty set then

```suggestion
                               sources=set())
```

If you want a singleton set containing the empty string

```suggestion
                               sources={''})
```